src/platforms/javascript/common/configuration/tree-shaking/index.mdx

```diff
@@ -79,3 +85,46 @@ const nextConfig = {
 For more information on custom webpack configurations in Next.js, see [Custom Webpack Config](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config) in the Next.js docs.
 
 </PlatformSection>
+
+## Tree Shaking Default Integrations
+
+By default, the Sentry SDK sets up a list of [default integrations](../integrations/default) that extend your
+SDK functionality. Furthermore, you can also add [additional](../integrations/plugin) or [custom](../integrations/custom)
```

Comment:
"Furthermore" is a bit formal, but I couldn't find a good replacement for it that didn't either sound as formal or create repetition. Suggest just taking it out.

```suggestion
SDK functionality. You can also add [additional](../integrations/plugin) or [custom](../integrations/custom)
```